static/src/javascripts/projects/common/modules/experiments/tests/glabs-traffic-driver.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import { trackAdRender } from 'commercial/modules/dfp/track-ad-render';
+
+const success = complete => trackAdRender('dfp-ad--glabs-left').then(complete);
+
+export const glabsTrafficDriver: ABTest = {
+    id: 'GlabsTrafficDriver',
+    start: '2017-09-08',
+    expiry: '2017-09-25',
```

Comment:
When I updated the dates I set it so that the JS would stop running on Monday 25th (at 23:59). This means the JS will silently stop the day before the Switch expires (I tend to use calendar reminders to know when to turn off tests).

If we want the test to run for two weeks, it may be worth bumping the dates slightly?

Reply:
Yep good point, I'll bump the dates now!

Reply:
:+1: